sdk/src/types/block/payload/signed_transaction/transaction.rs

```diff
@@ -439,6 +444,30 @@ fn verify_outputs(outputs: &[Output], visitor: &ProtocolParameters) -> Result<()
     Ok(())
 }
 
+fn verify_transaction_packable(transaction: &Transaction, _: &ProtocolParameters) -> Result<(), PayloadError> {
```

Comment:
`_: &ProtocolParameters` is needed here because it's needed as visitor for the outputs?

Reply:
yes

Reply:
Thanks